spring-integration-hazelcast/src/test/resources/HazelcastOutboundChannelAdapterTests-context.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:int="http://www.springframework.org/schema/integration"
+  xmlns:int-hazelcast="http://www.springframework.org/schema/integration/hazelcast"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/integration
+    http://www.springframework.org/schema/integration/spring-integration.xsd
+    http://www.springframework.org/schema/integration/hazelcast
+	classpath:org/springframework/integration/hazelcast/config/xml/spring-integration-hazelcast-4.2.xsd">
+
+	<int:channel id="mapChannel"/>
+	<int:channel id="listChannel"/>
+	<int:channel id="setChannel"/>
+	<int:channel id="queueChannel"/>
+	
+	<int-hazelcast:outbound-channel-adapter id="mapChannel" cache="distributedMap" />

```

Comment:
Right now you have ambiguity with this `id` and that `channel` definition. Not sure if it is an issue with your parser, but the `channel auto-creation` is really a feature of Spring Integration.